data/scripts/talkactions/god/reload.lua

```diff
@@ -49,19 +49,17 @@ function reload.onSay(player, words, param)
 	logCommand(player, "/reload", param)
 
 	local reloadType = reloadTypes[param:lower()]
-	if reloadType then
-		saveServer()
-		SaveHirelings()
-
-		Game.reload(reloadType)
-		logger.info("Reloaded {}", param:lower())
-
-		player:sendTextMessage(MESSAGE_LOOK, string.format("Reloaded %s.", param:lower()))
-		player:sendTextMessage(MESSAGE_ADMINISTRADOR, "Server is saved. Now will reload configs!")
-	elseif not reloadType then
+	if not reloadType then
 		player:sendTextMessage(MESSAGE_EVENT_ADVANCE, "Reload type not found.")
-		logger.warn("[reload.onSay] - Reload type '{}' not found", param)
```

Comment:
acho interessante manter o logger.warn.

Reply:
a mensagem já retorna que o reload não existe, não há nenhuma necessidade de deixar o warning se a mensagem já é visual dentro do client.

Reply:
@omarcopires not every administrator is the same person who develops the server and will check the logs